product_docs/docs/pgd/5/cli/configuring_cli.mdx

```diff
@@ -3,33 +3,32 @@ title: "Configuring PGD CLI"
 navTitle: "Configuring PGD CLI"
 ---
 
-PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security) - or equivalent (for example, edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
+PGD CLI can be installed on any system that can connect to the PGD cluster. To use PGD CLI, you need a user with PGD superuser privileges or equivalent. The PGD user with superuser privileges is the [bdr_superuser role](../security). An example of an equivalent user is edb_admin on a BigAnimal distributed high-availability cluster.
 
 ## PGD CLI and database connection strings
 
-You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+You might not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster, which means that the PGD CLI connects when run. 
 
```

Comment:
I thought this last phrase (in the original as well) was a bit unclear. Would it be accurate to say "PGD CLI connects to the PGD cluster when it runs"?

Reply:
"is able to connect to the cluster when run".